### DISCUSSION TITLE
I want to install on z8102ax

### DISCUSSION THREAD

**Discussion by jason5413** [Ysurac/openmptcprouter#3117] [Q&A]
Can you help me compile the firmware for z8102ax-128m? I always make mistakes when compiling here

**Comment by Ysurac** [comment]
There is 2 versions of z8102ax and the difference seems to be due more to firmware version than real flash capacity.
To compile, you need to follow https://github.com/Ysurac/openmptcprouter/wiki/Create-image-for-unsupported-platform but you need to replace "master" by "develop" as it's not yet in master tree.

**Reply by jason5413** [reply]
make[3]: *** [Makefile:70: /home/hik/openmptcprouter/z8102ax_128m/5.4/source/dl/mlvpn-8aa1b16.tar.xz] Error 128
make[3]: Leaving directory '/home/hik/openmptcprouter/feeds/openmptcprouter/mlvpn'
time: package/feeds/openmptcprouter/mlvpn/compile#0.25#0.13#10.64
    ERROR: package/feeds/openmptcprouter/mlvpn failed to build.
make[2]: *** [package/Makefile:116: package/feeds/openmptcprouter/mlvpn/compile] Error 1
make[2]: Leaving directory '/home/hik/openmptcprouter/z8102ax_128m/5.4/source'
make[1]: *** [package/Makefile:110: /home/hik/openmptcprouter/z8102ax_128m/5.4/source/staging_dir/target-aarch64_cortex-a53_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/hik/openmptcprouter/z8102ax_128m/5.4/source'
make: *** [/home/hik/openmptcprouter/z8102ax_128m/5.4/source/include/toplevel.mk:230：world] Error 2

**Reply by Ysurac** [reply]
z8102ax is not supported on 5.4 kernel. You must add OMR_KERNEL="6.1" to the build.sh command line (I've just added the variable in the doc).

**Reply by jason5413** [reply]
Thank you, I'll try again

**Reply by jason5413** [reply]
Hi，I can't find the management IP after flashing the firmware, how to solve it?
![Uploading 20240223181930.png…]()

**Reply by Ysurac** [reply]
192.168.100.1 on WAN port
